tests/ClientTest.php

```diff
@@ -2528,9 +2516,7 @@ public function testOurSerializer($after_creation)
                     ],
                 ]
             );
-            $events = $client->getSentEvents();
-            $event = array_pop($events);
-            $this->assertEquals(Dummy_Serializer::$default_value, $event['extra']['foo']);
+            $serializer->__phpunit_verify();
```

Comment:
This is wrong. There is no need to call an internal PHPUnit function to assert that a given method has been called. You should mock the methods and then just call the code as you would in the real use-case and PHPUnit will do the rest for you.